### ISSUE TITLE
SendKeyToElement does not throw exception if AndroidElement.setText() failed.

### ISSUE THREAD

**Issue by vmaxim** [appium/appium-uiautomator2-server#77]
#### logcat:
> 09-15 21:09:06.292  4571  4597 D appium  : Sending plain text to element: 00
> 09-15 21:09:06.294  4571  4597 W UiObject2: AccessibilityNodeInfo#performAction(ACTION_SET_TEXT) failed

#### uiautomator2:
[SendKeysToElement.safeHandle()](https://github.com/appium/appium-uiautomator2-server/blob/8eb809776b2a48435ec8574e28a553279124dd90/app/src/main/java/io/appium/uiautomator2/handler/SendKeysToElement.java#L83)

#### uiautomator:
[SetText.execute()](https://github.com/appium/appium-android-bootstrap/blob/master/bootstrap/src/io/appium/android/bootstrap/handler/SetText.java#L87)
